rma_reason_code/models/reason_code.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from random import randint
+
+from odoo import fields, models
+
+
+class RMAReasonCode(models.Model):
+    _name = "rma.reason.code"
+    _description = "RMA Reason Code"
+
+    def _get_default_color(self):
+        return randint(1, 11)
+
+    name = fields.Char("Code", required=True)
+    description = fields.Text()
+    type = fields.Selection(
```

Comment:
why not this? 
```suggestion
    something_type = fields.Selection(
```
it's bad trend to use the type field, as already exists [type](https://docs.python.org/3/library/functions.html#type) python function

Reply:
done!